src/main/java/de/csbdresden/stardist/StarDist2DBase.java

```diff
@@ -88,30 +88,38 @@ protected void export(String outputType, Candidates polygons, int framePosition)
     
     protected void exportROIs(Candidates polygons, int framePosition) {
         if (roiManager == null) {
-            IJ.run("ROI Manager...", "");
-            roiManager = RoiManager.getInstance();
+            roiManager = RoiManager.getRoiManager();
             roiManager.reset(); // clear all rois
         }
+
+        // Setting the RoiManager to invisible, the position of the ROI will be properly saved.
+        // The issue is in RoiManager.addRoi(), https://github.com/imagej/imagej1/blob/c4950ee1f19a25828e5ac915ef3f74e5aa13a6e2/ij/plugin/frame/RoiManager.java#L419
+        roiManager.setVisible(false);
+
         for (final int i : polygons.getWinner()) {
             final PolygonRoi polyRoi = Utils.toPolygonRoi(polygons.getPolygon(i));
-            // if (framePosition > 0) polyRoi.setPosition(0, 0, framePosition);
-            if (framePosition > 0) polyRoi.setPosition(framePosition);
+            if (framePosition > 0) polyRoi.setPosition(1, 1, framePosition);
```

Comment:
Why did you set `channel` and `slice` to 1 instead of 0?
([`Roi.setPosition`](https://github.com/imagej/ImageJA/blob/b78e7ca95ee93609fef29496e3d9e557314ef2e2/src/main/java/ij/gui/Roi.java#L2037-L2046) sets them to 0 if they're invalid.)

Reply:
You can see some tests I did here:

https://gist.github.com/lacan/2e37ec534d2e9e250e94486a8ebc8fa9
Basically, setPosition is linked to the ImagePlus methods, which are one-based (historical reasons)
https://imagej.nih.gov/ij/developer/api/ij/ImagePlus.html#convertIndexToPosition-int-
https://imagej.nih.gov/ij/developer/api/ij/ImagePlus.html#getStackIndex-int-int-int-

Reply:
And basically, even though is sets it to 0, that is inconsistent with the methods above so forcing yourself to use 1-based indexing helps for consistency...

Reply:
> (Roi.setPosition sets them to 0 if they're invalid.)
It does set it to 0 if it is negative. But as there are no axis limits bound to the ROI, you can set the position to anything you want, so long as it's not negative. And whoever did that bit did not know it was 1-indexed for historical reasons and to be compatible with the rest of ImageJ... A mess, I know..

Reply:
> A mess, I know..

Indeed.